extension/src/test/suite/setup/index.test.ts

```diff
@@ -1332,6 +1336,68 @@ suite('Setup Test Suite', () => {
       )
     })
 
+    it("should handle a message from the webview to update the user's self hosted url", async () => {
+      const { setup, mockExecuteCommand } = buildSetup({
+        disposer: disposable
+      })
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      stub(Setup.prototype as any, 'getCliCompatible').returns(true)
```

Comment:
[C] Maybe this should be moved into `buildSetup`

Reply:
and should be stubbed on the instance (if possible)

Reply:
I was thinking about that as well since we are running this line in multiple tests. I'll look into moving it in a followup.